src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -4195,3 +4195,112 @@ bool os::start_debugging(char *buf, int buflen) {
   }
   return yes;
 }
+
+// Java thread:
```

Comment:
Is this block pulled unchanged from the x86 specific file or has it been changed as well?

Reply:
it's a copy and paste from x86. bsd_zero also contains these functions (but slightly different)